metricbeat/module/mongodb/mongodb_test.go

```diff
@@ -77,7 +77,7 @@ func TestParseMongoURL(t *testing.T) {
 		},
 		{
 			Name:     "with options",
-			URL:      "mongodb://localhost:40001?connect=direct&authSource=me",
```

Comment:
`connect=direct` is a deprecated URI option but not yet removed. I think we should keep this test case and not replace it with the recommended URI option.

Reply:
Using deprecated connection strings in test cases is not considered as best practice as it can result to false positives or negatives.